src/service/extensions/tatumsdk.wallet.providers.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ITatumSdkContainer } from "./tatumsdk.container";
+import { TatumSdkExtension } from "./tatumsdk.extensions.dto";
+
+export abstract class TatumSdkWalletProviders extends TatumSdkExtension {
```

Comment:
Why just call it with prefix 'TatumSdk'? We know in which context we are, we know it is Sdk. We know it is Tatum repo. 
I would suggest to have naming just WalletProviders.

Reply:
It's a class that anyone from community developing a wallet provider will have to extend - it's for external use